src/anndata_to_avro.py

```diff
@@ -209,8 +209,17 @@ def default(self, obj):
             return json.JSONEncoder.default(self, obj)
 
     def ingest_generator():
+        # Some `uns` metadata has extremely large values that frustrate extract. Cap the allowed size and substitute
+        # a `None` value in these cases.
+        metadata_limit = 2 ** 20
+        uns = {}
+        for k, uncapped_v in adata.uns.data.items():
+            j = json.dumps(uncapped_v, cls=NumpyEncoder)
+            v = None if len(j) > metadata_limit else uncapped_v
```

Comment:
Would it be worth logging the type of metadata (is that available) that you are capping so that we understand if there are certain types that we can just exclude (rather than exclude purely by size)